src/data/complex/is_R_or_C.lean

```diff
@@ -896,4 +896,34 @@ end
 
 end linear_maps
 
+section nnreal
+open_locale nnreal
+
+/-- The natural coercion from `ℝ≥0` to `K` as a continuous `ℝ≥0`-linear map. -/
+noncomputable def of_nnreal_clm : ℝ≥0 →L[ℝ≥0] K :=
+{ to_fun := algebra_map ℝ≥0 K,
+  cont := is_R_or_C.continuous_of_real.comp continuous_induced_dom,
+  .. (algebra.of_id ℝ≥0 K).to_linear_map }
+
+@[simp] lemma of_nnreal_clm_coe : (of_nnreal_clm : ℝ≥0 → K) = algebra_map ℝ≥0 K := rfl
+
+@[continuity] lemma continuous_of_nnreal : continuous (coe : ℝ≥0 → K) :=
+map_continuous (of_nnreal_clm : ℝ≥0 →L[ℝ≥0] K)
+
+/-- The natural coercion from `ℝ≥0` to `K` as a continuous map. -/
+noncomputable def of_nnreal_cm : C(ℝ≥0, K) := ⟨algebra_map ℝ≥0 K, continuous_of_nnreal⟩
```

Comment:
Is this needed? `continuous_linear_map` has a `continuous_map_class`, so hopefully you can just use `of_nnreal_clm` where you need a `continuous_map`? The `_cm` naming is also nonexistent in mathlib.

Reply:
*Needed* might be a bit of a stronger word, but much more *convenient* is certainly true, especially for many of the use cases I envision. For clarity:
1. You have to write `((of_nnreal_clm : ℝ≥0 →L[ℝ≥0∞] K) : C(ℝ≥0, K))` to get it to coerce properly and not error out with "invalid type ascription".
2. These coercions don't play particularly well with compositions, and this `of_nnreal_cm` will be used often in compositions (e.g., for the continuous functional calculus, but also for some work I'm currently doing #16675 on ideals of continuous maps).

If you would prefer I spell it `of_nnreal_continuous_map` I can do that, but it's rather long. Also note that some other things in this file are abbreviated with things that I don't think are particularly common (e.g., `is_R_or_C.of_real_am`).